_includes/configuration/content-langs.md

```diff
@@ -0,0 +1,69 @@
+## `content_langs`
+
+{{ site.requires_5_9v }}
+
+This option allows you to override the languages displayed in the `language` dropdown toolbar button and the `language` menu item. This option accepts an array of language objects.
```

Comment:
We shouldn't be using "you" here. See https://www.tiny.cloud/docs/configure/contributing-docs/#pronouns